src/main/java/io/appium/java_client/service/local/AppiumServiceBuilder.java

```diff
@@ -296,7 +317,46 @@ private void loadPathToMainScript() {
         this.appiumJS = findMainScript();
     }
 
+    private String parseCapabilitiesIfWindows() {
```

Comment:
This method name does not do any check for the current platform, so I find it confusing it has `ifWindows` in its name

Reply:
rename to ...for Windows?!

Reply:
how about `capabilitiesToQuotedCmdlineArg` ?

Reply:
sounds good aswell...changed